musicround/routes/rounds.py

```
@@ -622,84 +625,45 @@ def send_email(round_id):
-            current_app.logger.info(f"Email sent successfully from {mail_sender} to {mail_recipient}")
-        
-        success_msg = f'Email sent successfully to {mail_recipient}!'
+        success_msg = message
         if request.headers.get('X-Requested-With') == 'XMLHttpRequest':
             return jsonify({'success': True, 'message': success_msg})
         else:
             flash(success_msg, 'success')
             return redirect(url_for('rounds.round_detail', round_id=round_id))
             
-    except smtplib.SMTPException as e:
+    except Exception as e:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)